src/pages/mission/goal/DailyGoalAchievement.tsx

```
@@ -0,0 +1,137 @@
+              <IcExpand className="h-24 w-24" />
+            )}
+          </button>
+
+          {isDetailOpen && (
+            <div className="px-20 pb-17">
+              <div className="mb-16 border-moamoa-300 border-t border-dashed" />
+
+              {questionResults.map((result, index) => (
+                <div
+                  key={index}
+                  className="flex items-center justify-between py-8 pl-10"
```

Biome (2.3.13)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)